src/index.js

```diff
@@ -0,0 +1,22 @@
+import './style.css';
+import List from './modules/list.js';
+
+const todoList = new List();
+todoList.display();
+// This line of code adds new Activity
+document.querySelector('#add-task').addEventListener('submit', (e) => {
+  e.preventDefault();
+  const activity = e.target.elements.activity.value;
+  todoList.addActivity(activity);
+  e.target.reset();
+});
+
+// This line of clears completed Activities
+document.querySelector('.clear-completed').addEventListener('click', () => {
+  todoList.clearCompleted();
+});
+
+// This line of code clears all handler
+document.querySelector('#delete-all').addEventListener('click', () => {
+  todoList.clearAll();
+});
```

Comment:
Please could you explain the difference between these 2 functions? Are they both related to the project's requirements or is the clearAll function an additional future? If both have the same function, maybe you should consider keeping one for the code to be more DRY.